services/subscription.go

```diff
@@ -409,11 +427,23 @@ func (le InitiatorSubscriptionLogEvent) ContractPayment() (*assets.Link, error)
 	return payment, nil
 }
 
+// Requester pulls the requesting address out of the LogEvent's topics.
+func (le InitiatorSubscriptionLogEvent) Requester() common.Address {
+	b := le.Log.Topics[RunLogTopicRequester].Bytes()
+	return common.BytesToAddress(b)
```

Comment:
Hmm, does this mean the address is being stores in the first 20 bytes, then the remaining 12 bytes are 0s? I was under the impression the 0s are the higher endian bytes.

Reply:
You're right that the higher order bytes the zeroes. [BytesToAddress takes the lower endian bytes.](https://github.com/ethereum/go-ethereum/blob/master/common/types.go#L231)